diff -r 8e576352f831 src/share/vm/utilities/ostream.cpp
--- a/src/share/vm/utilities/ostream.cpp	Tue Oct 07 15:44:11 2014 -0700
+++ b/src/share/vm/utilities/ostream.cpp	Thu Oct 09 08:52:46 2014 +0100
@@ -846,6 +846,7 @@
              os::current_process_id(), (int64_t)time_ms);
     // Write VM version header immediately.
     xs->head("vm_version");
+    xs->print_cr("<TweakVM/>");
     xs->head("name"); xs->text("%s", VM_Version::vm_name()); xs->cr();
     xs->tail("name");
     xs->head("release"); xs->text("%s", VM_Version::vm_release()); xs->cr();
