diff -r 6085f9633d8c common/autoconf/version-numbers
--- a/common/autoconf/version-numbers	Mon Oct 06 14:37:54 2014 +0200
+++ b/common/autoconf/version-numbers	Tue Oct 07 14:17:50 2014 +0100
@@ -31,7 +31,7 @@
 PRODUCT_NAME=OpenJDK
 PRODUCT_SUFFIX="Runtime Environment"
 JDK_RC_PLATFORM_NAME=Platform
-COMPANY_NAME=N/A
+COMPANY_NAME=JITWatch
 
 # Might need better names for these
 MACOSX_BUNDLE_NAME_BASE="OpenJDK"
